packages/main/src/plugin/image-registry.ts

```diff
@@ -438,6 +438,21 @@ export class ImageRegistry {
     };
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  async getManifestFromImageName(imageName: string): Promise<any> {
```

Comment:
we have the method getImageConfigLabels reusing the exact same lines
 
https://github.com/podman-desktop/podman-desktop/blob/ef79070fae759117fc1a961c42840eccffa1f760/packages/main/src/plugin/image-registry.ts#L457-L468

so this method should probably be refactored first calling getManifestFromImageName

Reply:
I'm not sure to follow, if the method returns also token and imageData parameters then I don't see any issue ?

```
const {imageData, token,manifest} = getManifestFromImageName(...)
```